components/director/pkg/graphql/schema.graphql

```diff
@@ -315,6 +360,38 @@ type APIDefinitionPage implements Pageable {
 	totalCount: Int!
 }
 
+type APIInstanceAuth {
```

Comment:
maybe `APIAuthPerRuntime` or `ApiAuthOnRuntime`?

Reply:
I agree that the APIInstanceAuth is not great, but `Runtime` in the name would be confusing. We will have auth per Service Instance, which means multiple auths on a single Runtime. The name with `Runtime` suggests otherwise.

I was thinking about Usage/Instance names, but I don't have any better alternative for now.